Fix compiling error due to elantech functions being present in both drivers/input/mouse/elantech.c and drivers/input/asusec/elan_i2c_asus.c
by removing elantech.o from Makefile for now. Needed for the Asus dock
--- a/drivers/input/mouse/Makefile
+++ b/drivers/input/mouse/Makefile
@@ -24,7 +24,7 @@
 psmouse-objs := psmouse-base.o synaptics.o
 
 psmouse-$(CONFIG_MOUSE_PS2_ALPS)	+= alps.o
-psmouse-$(CONFIG_MOUSE_PS2_ELANTECH)	+= elantech.o
+#psmouse-$(CONFIG_MOUSE_PS2_ELANTECH)	+= elantech.o
 psmouse-$(CONFIG_MOUSE_PS2_OLPC)	+= hgpk.o
 psmouse-$(CONFIG_MOUSE_PS2_LOGIPS2PP)	+= logips2pp.o
 psmouse-$(CONFIG_MOUSE_PS2_LIFEBOOK)	+= lifebook.o
